### ISSUE TITLE
Dashboards for training and evaluation loss

### ISSUE THREAD

**Issue by mseeger** [awslabs/keys_values#104]
**Is your feature request related to a problem? Please describe.**
There is currently no end-2-end support for tensorboard or other training/evaluation loss dashboards.

**Describe the solution you'd like**
Solution should show real-time metrics for training jobs running:
- Basic metrics (training loss, validation loss)
- Health of job (maybe including email warning if job fails)
- GPU memory usage

**Comment by mseeger** [comment]
Do this with AI
